### DISCUSSION TITLE
Unstylied Components

### DISCUSSION THREAD

**Discussion by q1b** [ayoayco/astro-reactive-library#251] [Ideas]
# Unstylied Components

## Unstylied Header Components
> Header is a common component for every website

There is one common functionalities attached to it, that motivated me, for this proposal.
- When you scroll down, ***header*** just get **hidden**, and when you scroll up header **appears**

**Comment by ayo-run** [comment]
Hi, @sacarvy thanks for opening this discussion. Which app are you referring to here? We have three in the repo: demo, docs, or landing page.
